autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -68,6 +69,15 @@ public void validateStopStartEntitlementAndDisableIfNotEntitled(Cluster cluster)
         }
     }
 
+    public void validateImpalaScheduleBasedScalingEntitlement(String clusterCrn, CloudPlatform cloudPlatform, String stackName) {
+        String accountId = Crn.safeFromString(clusterCrn).getAccountId();
+        List<String> missingEntitlements = entitlementValidationService.impalaScheduleBasedScalingMissingEntitlements(accountId, cloudPlatform);
+        if (!missingEntitlements.isEmpty()) {
+            throw new BadRequestException(messagesService.getMessage(MessageCode.IMPALA_SCHEDULE_BASED_SCALING_ENTITLEMENT_NOT_ENABLED,
+                    List.of(cloudPlatform.name(), accountId, missingEntitlements)));
```

Comment:
Same thing here. Don't throw an exception with the Entitlements in it - if this has a chance of reaching customers.
These can be logged instead.

Reply:
Removed